src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -105,6 +107,25 @@ limitations under the License.
             <include cluster="Power Source" client="false" server="true" clientLocked="true" serverLocked="true"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-electricalmeasurement</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Electrical Measurement</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0xFFF10010</deviceId><!-- TODO: ID-TBD -->
+        <class>Utility</class>
+        <scope>Node</scope>
+        <clusters lockOthers="true">
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
```

Comment:
Probably shouldn't be there; those are mandatory attributes anyway.

Reply:
Because someone did it and everyone else copy/pasted and got it rubber-stamped.  :(

Reply:
Removed.